stacks/root.yml

```diff
@@ -116,6 +120,8 @@ Mappings:
       prefix: 38
     Crier:
       prefix: 42
+    Metrics:
```

Comment:
Metrics should have a unique # here (jingle has already taken 38). Also good to maintain order in increasing listener priority. My guess is Metrics should be somewhere around -- 36? Lower priority than castle (which will get hit at least as often as metrics) but higher than jingle (which will only be the shows we sell ads for)

Reply:
done!

Reply:
last nitpicky thing -- good to maintain this list in order of the priority prefix, so just move this to below castle / above jingle and then i think all set!